src/main/java/profile_screens/ProfileScreen.java

```diff
@@ -17,55 +18,58 @@ public class ProfileScreen extends JFrame {
 
-        ReviewGateway reviewGateway = new ReviewGateway();
+            ReviewGateway reviewGateway = new ReviewGateway();
 
-        this.reviews = reviewGateway.getReviews(profile.getPast_reviews());
+            this.reviews = reviewGateway.getReviews(profile.getPast_reviews());
 
-        profile_screen_window = new JFrame();
+            profile_screen_window = new JFrame();
 
-        JLabel title = new JLabel(profile.getUsername());
-        title.setAlignmentX(Component.CENTER_ALIGNMENT);
+            JLabel title = new JLabel(profile.getUsername());
+            title.setAlignmentX(Component.CENTER_ALIGNMENT);
 
-        int totalStars = 0;
+            int totalStars = 0;
 
-        ArrayList<JPanel> review_panels = new ArrayList<>();
+            ArrayList<JPanel> review_panels = new ArrayList<>();
 
-        for (Review review : this.reviews) {
-            JLabel restarauntName = new JLabel(review.getRestaurant());
-            JLabel reviewText = new JLabel(review.getText());
-            int starCount = review.getStars();
-            JLabel stars = new JLabel(Integer.toString(starCount) + "/5 stars");
+            for (Review review : this.reviews) {
+                JLabel restarauntName = new JLabel(review.getRestaurant());
+                JLabel reviewText = new JLabel(review.getText());
+                int starCount = review.getStars();
+                JLabel stars = new JLabel(Integer.toString(starCount) + "/5 stars");
 
 
-            JPanel panel = new JPanel();
-            panel.add(restarauntName);
-            panel.add(stars);
-            panel.add(reviewText);
+                JPanel panel = new JPanel();
+                panel.add(restarauntName);
+                panel.add(stars);
+                panel.add(reviewText);
 
-            review_panels.add(panel);
+                review_panels.add(panel);
 
-            totalStars = totalStars + starCount;
-        }
-
-        int averageStars = totalStars / this.reviews.size();
+                totalStars = totalStars + starCount;
+            }
 
+            int averageStars = totalStars / this.reviews.size();
 
-        JLabel averageStarLabel = new JLabel(Integer.toString(averageStars));
 
-        profile_screen_window.add(title);
-        profile_screen_window.add(averageStarLabel);
+            JLabel averageStarLabel = new JLabel(Integer.toString(averageStars));
 
+            profile_screen_window.add(title);
+            profile_screen_window.add(averageStarLabel);
 
-        for (JPanel panel : review_panels) {
-            profile_screen_window.add(panel);
-        }
 
-        profile_screen_window.setVisible(true);
+            for (JPanel panel : review_panels) {
+                profile_screen_window.add(panel);
+            }
 
+            profile_screen_window.setVisible(true);
+        }catch(IOException e){
+            System.out.println("An error occurred. Please try again later.");
+        }
```

Comment:
Were these changes made by you?

Reply:
same comment as Tim, can't seem to trace these changes to any of Dave's commits